check-osp/check_osp_main.py

```diff
@@ -122,6 +133,15 @@ def main():
         if args.services:
```

Comment:
Instead of checking services and services_file args, why don't we check `args.measurement`? That way you can avoid the multiple calls to `measure_func`:

```python,
if args.measurement == "services_status":
    if args.services_file:
        get_services_dict
    else:
        get_services_dict
     call measurement func
```